unlock-app/src/components/interface/locks/Manage/elements/MetadataCard.tsx

```diff
@@ -350,9 +364,20 @@ export const MetadataCard = ({
 
   const ownerIsManager = owner?.toLowerCase() === manager?.toLowerCase()
   const showManager = !ownerIsManager && manager !== ADDRESS_ZERO
+  const canTransferKeyOwnership =
+    isLockManager && isTransferAllowed && !isLoadingTransferFee
```

Comment:
make sure that transfer is visible only if the key is transferable & the user is the lock manager